test/unit/styling-build-adopted-stylesheets.html

```diff
@@ -0,0 +1,1027 @@
+<!doctype html>
```

Comment:
Is this just `styling-scoped.html`, with the flag turned on and `@apply` removed?

Reply:
Yes, with the addition of one test that checks `adoptedStyleSheets`